pytest-embedded-idf/pytest_embedded_idf/serial.py

```diff
@@ -161,6 +162,8 @@ def flash(self, app: Optional[IdfApp] = None) -> None:
                 else:
                     _args.append(str(v))
 
+        if '--baud' not in _args:
+            _args.extend(['--baud', os.getenv('ESPBAUD', '1000000')])
         _args.append('write_flash')
```

Comment:
I'll merge it as is. next time better to add a blank line here, to make code blocks referring different logic separated.

```suggestion
            _args.extend(['--baud', os.getenv('ESPBAUD', '1000000')])
            
        _args.append('write_flash')
```